### PR TITLE
GPII-3394: more chart variable cleanups

### PR DESCRIPTION
**Pull request by mrtyler**
As Stepan noted in #233:

> LGTM (but if you catched everything I have no clue :D)

Turns out I missed something. :)

### CODE REVIEW

**Comment by natarajaya** [comment]
LGTM

File: shared/charts/gpii-preferences/values.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ dnsNames:
 - preferences.test.local
 
 ingress:
-  disable_ssl_redirect: false
+  disableSslRedirect: false
```
Comment (natarajaya) [inline shared/charts/gpii-preferences/values.yaml]: What was wrong with `disable_ssl_redirect`?

Comment (mrtyler) [reply shared/charts/gpii-preferences/values.yaml]: https://github.com/gpii-ops/gpii-infra/pull/230#discussion_r241529042

Comment (natarajaya) [reply shared/charts/gpii-preferences/values.yaml]: Got it, thanks!

**Pull request merged**